Src/BootCamp.Chapter/Calculate.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Calculate
```

Comment:
Horrible name.
1) should be a noun
2) should logically named. Even demo would be better. Maybe Calculator? Etc...

Reply:
Agreed - change to be made.